content/docs/examples/codelabs-tutorials.md

```diff
@@ -1,41 +1,42 @@
 +++
 title = "Codelabs, Workshops, and Tutorials"
-description = "Recommended end-to-end tutorials, workshops, walk-throughs, and codelabs"
+description = "Recommended end-to-end tutorials, workshops, walk throughs, and codelabs"
 weight = 20
 +++
 
-{{% blocks/content-item title="OpenShift Kubeflow Workshop"
-  url="https://github.com/AICoE/openshift_kubeflow_workshop" %}}
-Run Kubeflow on Red Hat [OpenShift](https://www.openshift.com/).
-{{% /blocks/content-item %}}
+## Agile Stacks Kubeflow Pipelines tutorials
 
-{{% blocks/content-item title="Agile Stacks Kubeflow Pipelines Tutorial"
-  url="https://www.agilestacks.com/tutorials/ml-pipelines" %}}
 Run Kubeflow Pipelines tutorials on AWS, GCP, or on-prem hardware using [Agile Stacks](https://www.agilestacks.com/).
 Pipeline templates provide step-by-step examples for working with object storage filesystem, Kaniko, Keras, and Seldon.
-{{% /blocks/content-item %}}
-
-{{% blocks/content-item title="Katacoda scenarios"
-  url="https://www.katacoda.com/kubeflow" %}}
-Follow the tutorials to deploy Kubeflow and run a machine learning model.
-{{% /blocks/content-item %}}
-
-{{% blocks/content-item title="Introduction to Kubeflow Codelab"
-  url="https://codelabs.developers.google.com/codelabs/kubeflow-introduction/index.html" %}}
-Run MNIST with Kubeflow on Google Kubernetes Engine.
-{{% /blocks/content-item %}}
-
-{{% blocks/content-item title="Introduction to Kubeflow Qwiklab"
-  url="https://www.qwiklabs.com/catalog_lab/933" %}}
-Run MNIST with resources provided by Qwiklabs.
-{{% /blocks/content-item %}}
-
-{{% blocks/content-item title="Kubeflow End to End Codelab"
-  url="https://codelabs.developers.google.com/codelabs/cloud-kubeflow-e2e-gis/index.html" %}}
-Run GitHub Issue Summarization with Kubeflow on Google Kubernetes Engine.
-{{% /blocks/content-item %}}
-
-{{% blocks/content-item title="Kubeflow End to End Qwiklab"
-  url="https://www.qwiklabs.com/catalog_lab/1046" %}}
-Run GitHub Issue Summarization with resources provided by Qwiklabs.
-{{% /blocks/content-item %}}
+
+* [ML Pipeline Templates: End-to-end Tutorial](https://www.agilestacks.com/tutorials/ml-pipelines).
+
+## Google codelabs
```

Comment:
Since the proper name is Google Developers Codelabs, should that be used here in the title? Just wondering.